backend/src/main/java/gov/cdc/usds/simplereport/db/repository/ApiUserRepository.java

```diff
@@ -25,6 +25,10 @@
   @Query(BASE_QUERY + " and loginEmail = :email")
   public Optional<ApiUser> findByLoginEmail(String email);
 
+  // This query is run on user-supplied input so needs to be normalized
+  @Query("FROM #{#entityName} e WHERE lower(loginEmail) = lower(:email)")
+  public Optional<ApiUser> findByLoginEmailIncludeArchived(String email);
```

Comment:
i feel like it's not the responsibility of the DB layer to lower-case the input--feel like that should happen at the service layer.